docs/docs/Components/components-processing.mdx

```diff
@@ -13,7 +13,7 @@ They have many uses, including:
 
 * Feed instructions and context to your LLMs and agents with the [**Prompt Template** component](#prompt-template).
 * Extract content from larger chunks of data with a [**Parser** component](#parser).
-* Filter data with natural language with the [**Smart Function** component](#smart-function).
+* Filter data with natural language with the [**Smart Transform** component](#smart-function).
```

Comment:
please find and replace all `#smart-function` with `#smart-transform` since the section name is now "smart transform"